fastlane/Constants/Secret.swift

```diff
@@ -12,4 +12,6 @@ enum Secret {
     static let keychainPassword: String = "<#keychainPassword#>"
 
     static let firebaseCLIToken: String = "<#firebaseCLIToken#>"
+
+    static let appstoreConnectAPIKey = "<#appstoreConnectAPIKey#>"
```

Comment:
This is the only constant without `: String`. I think we don't need the `: String` at all for these cases?

Reply:
@blyscuit Updated as this [thread](https://nimble-co.slack.com/archives/C418DBL2W/p1664364433445829) too in [f419874](https://github.com/nimblehq/ios-templates/pull/359/commits/f419874843448853c24199fb176948596b703a8b) as